lms/templates/fields/field_textarea.underscore

```diff
@@ -13,10 +17,12 @@
         var textareaDescribedBy = (message ? 'u-field-message-help-' : 'u-field-placeholder-value-') + id;
         if (mode === 'edit') {%>
             <textarea id="u-field-textarea-<%- id %>" rows="4" aria-describedby="<%- textareaDescribedBy %>"><%- value %></textarea>
-        <% } else {
-            %><a href="#"><span class="sr"><%- screenReaderTitle %></span><span class="u-field-value-readonly" aria-hidden="false" aria-describedby="u-field-placeholder-value-<%- id %>"><%- value %></span><span class="sr"><%- gettext('Click to edit') %></span></a><%
-        }
-        %><span class="sr" id="u-field-placeholder-value-<%- id %>"><%- placeholderValue %></span>
+        <% } else if (editable === 'never') { %>
+            <span class="sr"><%- screenReaderTitle %></span><span class="u-field-value-readonly" aria-hidden="false"><%- value %></span>
```

Comment:
So, this is already labelled by the ```u-field-title``` above.  Adding a ```sr``` only label here is redundant.  Because of the additional aria-labelledby on the container, the words About me are read 3 times to the user.